hpat/tests/test_numba.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+import hpat
+import numba
+from hpat import *
+from numba.typed import Dict
+from collections import defaultdict
+
+class TestNumba(unittest.TestCase):
```

Comment:
Could you please rename it to something like `HPAT_Jit_issues`?

Reply:
Renamed